src/main/java/reposense/util/TimeUtil.java

```diff
@@ -21,7 +23,7 @@
     private static Long startTime;
     private static final String DATE_FORMAT_REGEX =
             "^((0[1-9]|[12][0-9]|3[01])\\/(0[1-9]|1[012])\\/(19|2[0-9])[0-9]{2})";
-    private static final DateFormat CLI_ARGS_DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
+    private static final DateTimeFormatter CLI_ARGS_DATE_FORMAT = DateTimeFormatter.ofPattern("d/M/uuuu HH:mm:ss");
```

Comment:
I can change all the other `uuuu` but changing this one back seems to break some tests.

Reply:
For this one, `yyyy` doesn't work with `ResolverStyle.STRICT` as described [here](https://stackoverflow.com/questions/27454025/unable-to-obtain-localdatetime-from-temporalaccessor-when-parsing-localdatetime). 

It is possible to use `yyyy` with the default `ResolverStyle` but that would break the invalid date test in which "31 February" is used as the input date.

Reply:
I see, thanks for the linked StackOverflow answer! Let's use `uuuu` as you suggested.